### DISCUSSION TITLE
Skip PR creation for newer versions with same digest

### DISCUSSION THREAD

**Discussion by pascal-hofmann** [renovatebot/renovate#30501] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitHub Enterprise 3.12.7, Renovate 37.440.7

### Please tell us more about your question or problem

Hi,
we have a repository that contains references to the [gethue docker image](https://hub.docker.com/r/gethue/hue/tags). This image gets new tags each day, but the digest does not change very often.

Is there a way to tell renovate to only create new PRs if the digest changes (so only in case there is really something new)?

Thanks in advance
Pascal

**Comment by rarkins** [comment, marked as answer]
I'm not aware of any way to achieve this today
